setup.py

```diff
@@ -19,7 +19,7 @@
         'shapely>=1.6b3'
     ],
     include_package_data=True,
-    data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
+    # data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
```

Comment:
@jbouffard Is this the thing that included the jar in the wheel so it can be called in the vanilla case of "import geopyspark" in python process ?